torch/_inductor/mkldnn_lowerings.py

```diff
@@ -508,6 +508,18 @@ def qlinear_unary(
             algorithm,
             layout=None,
         ):
+            # Symmetrically int8 Quantized activation. In case of asymmetrically int8 quantized activation,
+            # we'd have to compute compensation corresponding to the zero-point of the activation.
+            # If activation is of uint8 dtype, compensation is applied even in case of symmetric quantization.
+            # because the zero-points are non-zero in that case.
+            sym_quantized_int8_act = False
```

Comment:
Can we define a function to init these flags here instead of change its value everywhere?